docs/tools/swift.pod

```diff
@@ -1,30 +1,114 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift - General purpose programming language
 
 =head1 SYNOPSIS
 
-B<swift> [-help] [I<input-filename> [I<program-arguments>]]
+To invoke the Swift REPL (Read-Eval-Print-Loop):
+
+=over
+
+B<swift>
+
+=back
+
+To execute a Swift program:
+
+=over
+
+B<swift> program.swift -- <arguments>
+
+=back
+
+To work with the Swift Package Manager:
+
+=over
+
+B<swift> build|package|run|test [options]
+
+=back
+
+To invoke the Swift Compiler:
+
+=over
+
+B<swiftc> [options] <inputs>
+
+=back
+
+The list of supported options is available through the "-help" option:
```

Comment:
I'm being very nitpicky, but:

The original text was: "The **full** list of supported options..." If you're going to drop "full," then probably best if it's simply "**A** list of supported options..." (or, restore the original).